stac_geoparquet/arrow/_to_parquet.py

```diff
@@ -141,7 +141,10 @@ def create_geoparquet_metadata(
             "crs": None,
         }
 
-    return {b"geo": json.dumps(geo_meta).encode("utf-8")}
+    return {
+        b"geo": json.dumps(geo_meta).encode("utf-8"),
+        b"stac:geoparquet_version": b"1.0.0",
```

Comment:
Probably want this to be a module-level constant somewhere in the package, for easy future updating?